experimental/FTheoryTools/src/TateModels/methods.jl

```diff
@@ -0,0 +1,61 @@
+#####################################################
+# 1: Fiber analysis
+#####################################################
```

Comment:
This has been moved from another file, right?

It seems that the code which has been moved is not tested. Any specific reason why this is the case? I.e. are the tests, for example, disabled because they run too long?